src/webparts/LaserficheRepositoryAccessWebPart/components/LaserficheRepositoryAccessWebPart.tsx

```diff
@@ -65,18 +65,16 @@ export default class LaserficheRepositoryAccessWebPart extends React.Component<
   public lfRepoTreeService: LfRepoTreeNodeService;
   public lfFieldsService: LfFieldsService;
   public showFieldContainer: boolean = false;
-  public REGIONAL_DOMAIN: string = `${this.props.region}`/*  "laserfiche.com" */; // only update this if you are using a different region or environment
 
   constructor(props: ILaserficheRepositoryAccessWebPartProps, state: ILaserficheRepositoryAccessWebPartState) {
     super(props);
-    SPComponentLoader.loadCss('https://cdn.jsdelivr.net/npm/@laserfiche/lf-ui-components@13/cdn/indigo-pink.css');
-    SPComponentLoader.loadCss('https://cdn.jsdelivr.net/npm/@laserfiche/lf-ui-components@13/cdn/lf-ms-office-lite.css');
+    SPComponentLoader.loadCss("https://cdn.jsdelivr.net/npm/@laserfiche/lf-ui-components@13/cdn/indigo-pink.css");
```

Comment:
use lfxstatic url everywhere
replace cdn.jsdelivr

Reply:
I will do this in a future PR